apps/council-ui/pages/proposals/details.tsx

```diff
@@ -94,18 +94,22 @@ export default function ProposalPage(): ReactElement {
     return <ErrorMessage error={error} />;
   }
 
-  const proposalTitle = data?.title ?? `Proposal ${id}`;
+  const proposalTitle = getProposalTitle(data, status, id);
```

Comment:
Since we're already checking the status and rendering a skeleton below, we don't need the function to return a skeleton.

This could stay the way it was without a fn and we just use the status in the markup below.